docs/csharp/programming-guide/classes-and-structs/how-to-implement-a-lightweight-class-with-auto-implemented-properties.md

```diff
@@ -13,12 +13,15 @@ ms.assetid: 1dc5a8ad-a4f7-4f32-8506-3fc6d8c8bfed
 
 This example shows how to create an immutable lightweight class that serves only to encapsulate a set of auto-implemented properties. Use this kind of construct instead of a struct when you must use reference type semantics.
 
-You can make an immutable property in two ways:
+You can make an immutable property in the following ways:
 
-- You can declare the [set](../../language-reference/keywords/set.md) accessor to be [private](../../language-reference/keywords/private.md).  The property is only settable within the type, but it is immutable to consumers.
+- Declare the [set](../../language-reference/keywords/set.md) accessor to be [private](../../language-reference/keywords/private.md).  The property is settable within the type, but it is immutable to consumers.
 
   When you declare a private `set` accessor, you cannot use an object initializer to initialize the property. You must use a constructor or a factory method.
```

Comment:
Here again, I'd move these two paragraphs below the recommendation to "Declare only the `get` accessor, ..."